### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jayunit100**
from my conversation with @marun  ... wanted to get this in writing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 [![Build Status](https://travis-ci.org/kubernetes-sigs/federation-v2.svg?branch=master)](https://travis-ci.org/kubernetes-sigs/federation-v2 "Travis")
 [![Image Repository on Quay](https://quay.io/repository/kubernetes-multicluster/federation-v2/status "Image Repository on Quay")](https://quay.io/repository/kubernetes-multicluster/federation-v2)
 
-## Summary
+## Summary of Federation V2 status
+
+Note, that we've changed quite a bit since V1, see the FAQ for salient details if your interested in planning a strategic direction around federation.
```
Comment (yamt) [inline README.md]: your -> you're

File: README.md [hunk 2]
```diff
@@ -64,3 +64,21 @@ interested in contributing.
 
 Participation in the Kubernetes community is governed by the
 [Kubernetes Code of Conduct](./code-of-conduct.md).
+
+## FAQ
+
+Some FAQ's about the future of federation and where its headed.
+
+### How do kuberenetes objects differe in federation V2, from standard objects ?
```
Comment (yamt) [inline README.md]: differe -> differ ?

File: README.md [hunk 3]
```diff
@@ -64,3 +64,21 @@ interested in contributing.
...
+
+- In the V1 implementation, federation actually federated core API objects across clusters.  This was very elegant
+but not scalable - all objects simply cant be federated in a transparent manner.
```
Comment (yamt) [inline README.md]: cant -> can't

File: README.md [hunk 4]
```diff
@@ -64,3 +64,21 @@ interested in contributing.
...
+
+- In V2, we move to a federated model where all cluster objects which are federated are a different type.  For example, there is a FederatedDeployment which is different from a Deployment.  we will provide tooling to automatically federate *certain* objects, and even possibly add support at the helm layer for federating entire namespaces in a user friendly manner.
```
Comment (yamt) [inline README.md]: maybe better to avoid a very long line

**Comment by marun** [comment]
@jayunit100 Thank you for doing this!  Modulo the grammar/spelling comments from @yamt, I'd like to see this merge so these details are captured in the repo.

**Comment by marun** [comment]
Closing.  I will be including the broad strokes of this PR in doc rework for the beta.

**Pull request closed without merging**